response_operations_ui/views/collection_exercise.py

```diff
@@ -61,10 +60,14 @@
     "collection_exercise_bp", __name__, static_folder="static", template_folder="templates"
 )
 
+CI_TABLE_TEXT = {
+    "EQ": {"no_instrument": "Add", "restricted": "View", "has_permission": "Select or Add"},
+    "SEFT": {"no_instrument": "Upload", "restricted": "View", "has_permission": "View or Upload"},
+}
 
-def build_collection_exercise_details(short_name: str, period: str, include_ci=True) -> dict:
-    """
 
+def build_collection_exercise_details(short_name: str, period: str, include_ci: bool = False) -> dict:
```

Comment:
include_ci=True was incorrectly being used, meaning most of the calls made to build_collection_exercise_details were processing cis unnecessarily